README.adoc

```diff
@@ -0,0 +1,478 @@
+= Spring Security Docs Home
```

Comment:
This reference to Spring Security should be updated to Batch. There are other occurrences as well in the readme. I can take of those when merging the PR.